python/mxnet/gluon/estimator/estimator.py

```diff
@@ -261,6 +254,8 @@ def fit(self, train_data,
         for epoch in range(self.max_epoch):
             # epoch begin
             self.current_epoch = epoch
+            # Number of samples trained after every batch
+            completed_samples = 0
```

Comment:
isn't this just batch_size?

Reply:
This variable stores the number of processed samples which is used in LoggingHandler for printing to console.